columbo/_interaction.py

```diff
@@ -39,60 +39,121 @@ def _or_default(value, default: T) -> T:
     return default if isinstance(value, _Sentinel) else value
 
 
-class Echo:
-    """Display a message to the user."""
+class Displayable(ABC):
```

Comment:
The base class for interactions that prompt the user is [exposed as part of the API](https://github.com/wayfair-incubator/columbo/blob/74265590f0bb57f6fd8f0c1b754d71e1d881fc8c/columbo/__init__.py#L15). Can you do that as well for `Displayable`.